mobile_config/src/mobile_radio_tracker.rs

```diff
@@ -77,6 +77,8 @@ struct TrackedMobileRadio {
     hash: String,
     last_changed_at: DateTime<Utc>,
     last_checked_at: DateTime<Utc>,
+    asserted_location: Option<i64>,
+    asserted_location_change_at: Option<DateTime<Utc>>,
```

Comment:
can we make this `asserted_location_changed_at` to keep inline with other fields

Reply:
done